src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/SaveSamlEntityCommand.php

```diff
@@ -487,4 +492,14 @@ public function setTypeOfService(array $typeOfServices): void
     {
         $this->typeOfService = $typeOfServices;
     }
+
+    public function isPublicInDashboard(): ?bool
+    {
+        return $this->isPublicInDashboard;
+    }
+
+    public function setIsPublicInDashboard(bool $isPublic): void
+    {
+        $this->isPublicInDashboard = $isPublic;
+    }
```

Comment:
Same, why not reutn false. Also, I see some repetition. Don;t know if that is okay, because code duplication is way cheaper than using the wrong abstractie - Sandy Metz

Reply:
See previous comment